notebooks/pokemon_classifier.ipynb

```diff
@@ -973,10 +963,9 @@
    "source": [
     "def predict_pokemon(id: str, verbose=0):\n",
     "    \n",
-    "    while len(id) < 3:\n",
-    "        id = \"0\" + id\n",
+    "    id = id.zfill(3)\n",
     "    \n",
-    "    img = tf.keras.utils.load_img(f'{PROJECT_ROOT}/{DESTINATION_DIRECTORY}/pokemon/{id}.png', color_mode='rgba')\n",
+    "    img = tf.keras.utils.load_img(PROJECT_ROOT/DESTINATION_DIRECTORY/'pokemon'/f'{id}.png', color_mode='rgba')\n",
```

Comment:
In general, it is better to use `pathlib.Path` for Paths. The reason is that if you use plain string for paths, then the folder separator does not automatically adjust to your OS. This means that the code would not work on Windows.